SmthJedi/license.txt

```diff
@@ -1 +1,11 @@
-These files are free to use, modify, and redistribute without attribution or any other restrictions.
\ No newline at end of file
+These files are free to use, modify, and redistribute without attribution or any other restrictions.
+
+Modifications by Brian Conner 2022:
+Detailed changes can be found in this font's ReadMe.txt.
+Any additional sounds sourced from creative-commons licensed freesound.org.
```

Comment:
Please don't say "any sounds added ...."
Please specify which sounds were added and fall under this license.

Reply:
Done.